### PR TITLE
fix 502 when exporting guest list

### PR DESCRIPTION
**Pull request by david-cahill**
ref https://github.com/CoderDojo/community-platform/issues/717

### CODE REVIEW

File: lib/events/cd-events.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ module.exports = function (options) {
     var locality = (args.req$ && args.req$.cookies.NG_TRANSLATE_LANG_KEY && args.req$.cookies.NG_TRANSLATE_LANG_KEY.replace(/\"/g, '')) || args.req$ && args.req$.headers['accept-language'];
     if (!locality) locality = 'en_US';
     locality = formatLocaleCode(locality);
-    var zenHostname = args.req$.headers.host || '127.0.0.1:8000';
```
Comment (ckiss) [inline lib/events/cd-events.js]: @david-cahill this is a variable initialization, don't think your fix will work.

Comment (david-cahill) [inline lib/events/cd-events.js]: @ckiss how so ? It checks that `args.req$ && args.req$.headers.host` exists and if not, applies `'127.0.0.1:8000'` as the hostname

Comment (ckiss) [inline lib/events/cd-events.js]: what if `args.req$ && args.req$.headers.host ` exists? shouldn't assign something in that case too?

Comment (ckiss) [inline lib/events/cd-events.js]: I think this should be something like this now: `args.req$ && args.req$.headers.host ? args.req$.headers.host : '127.0.0.1:8000'`

**Comment by ckiss** [comment]
LGTM :)

**Comment by tangentfairy** [comment]
:+1:

**Pull request merged**